src/javascript/app/pages/cashier/account_transfer.js

```diff
@@ -87,7 +87,7 @@ const AccountTransfer = (() => {
     };
 
     // TODO: change values when back-end updates logic
-    const getMinAmount = () => (isCryptocurrency(client_currency) ? 0.002 : 1);
+    const getMinAmount = () => (isCryptocurrency(client_currency) ? 0.004 : 1);
```

Comment:
are we updating for all crypto? or just btc?

Reply:
Yes. Only BTC for now